src/Tutor.Core/Domain/KnowledgeMastery/KnowledgeComponentMastery.cs

```diff
@@ -71,6 +71,16 @@ public Result AbandonSession()
         return SessionTracker.Abandon();
     }
 
+    public Result PauseSession()
+    {
+        return SessionTracker.Pause();
+    }
+
+    public Result ContinueSession()
+    {
+        return SessionTracker.Continue();
+    }
+
     private void JoinOrLaunchSession()
     {
```

Comment:
Should rephrase the comment at the bottom of this class (in private static void When(KnowledgeComponentEvent @event)) to cover all Session Lifecycle events. Instead of appending the Continue and Pause, set the comment to
`// No action for AssessmentItemSelected, InstructionalItemsSelected, SolutionRequest, InstructorMessageEvent, and SessionLifecycle events.`